server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -22,6 +22,7 @@ data:
         "lumberjack": {
             "options": {
                 "enableGlobalTelemetryContext": {{ .Values.lumberjack.options.enableGlobalTelemetryContext }}
+                "enableSanitization": {{ .Values.lumberjack.options.enableSanitization }}
```

Comment:
We might also need to make similar changes to the Historian and GitRest configmaps:

https://github.com/microsoft/FluidFramework/blob/main/server/charts/historian/templates/gitrest-configmap.yaml

https://github.com/microsoft/FluidFramework/blob/main/server/charts/historian/templates/historian-configmap.yaml

And their values.yaml:
https://github.com/microsoft/FluidFramework/blob/main/server/charts/historian/values.yaml

Reply:
Got it thanks! I have a question with gitrest and historian, do their configureLogging functions need to be modified as well? https://github.com/microsoft/FluidFramework/server/gitrest/packages/gitrest-base/src/logger.ts and https://github.com/microsoft/FluidFramework/server/historian/packages/historian-base/src/logger.ts

Reply:
Good question! Those 2 separate `configuringLogging` functions were created  as a workaround to some package versioning issues that caused trouble due to Lumberjack's being a singleton.

Recently, @znewton added some great improvements in our telemetry setup, including making Lumberjack a true global variable that would prevent this kind of issue (I think). Zach, does that mean that we can skip updating GitRest's and Historian's specific implementations of `configureLogging`?

Reply:
Spoke internally and indeed these should be made irrelevant with the global telemetry context.